django_evaluation/settings/local.py

```diff
@@ -310,6 +310,10 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
     "variable",
 ]
 MENU_ENTRIES = []
+
+# Sometimes it is desired to put a link into the navbar which does not correspond
+# to a template inside django. If something like this is needed, put a slash as a
+# prefix to your relative url (the second value in each of the lists), e.g. "/impressum"
```

Comment:
Do we have a docu for the web, I believe not - correct?

Reply:
Nope.

Reply:
K, but could add a an example into the README?

Reply:
For what? The Menu_entries? I mean, I could, but that would appear a bit random to me to explain this little thingy and leave everything else as it is. There are things in the web going on that are way more harder to grasp than this and are still not documented. If you want more documentation perhaps we should tackle that in a separate PR (and clean up the settings while we are at it).

Reply:
ok ok